### ISSUE TITLE
simpleruntimeeventvalidation test failing in CI

### ISSUE THREAD

**Issue by ericstj** [dotnet/runtime#88499]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=331166
Build error leg or test failing: tracing\\eventpipe\\simpleruntimeeventvalidation\\simpleruntimeeventvalidation\\simpleruntimeeventvalidation.cmd
Pull request: https://github.com/dotnet/runtime/pull/87562

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#how-to-fill-out-a-known-issue-error-section).

```json
{
  "ErrorMessage": "",
  "ErrorPattern": "tracing.eventpipe.simpleruntimeeventvalidation.simpleruntimeeventvalidation.simpleruntimeeventvalidation",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=331166
**Error message validated:** `tracing.eventpipe.simpleruntimeeventvalidation.simpleruntimeeventvalidation.simpleruntimeeventvalidation`
**Result validation: :white_check_mark:** Known issue matched with the provided build.
**Validation performed at:** 7/6/2023 11:55:11 PM UTC

### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[336390](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336390)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  linux x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336390/logs/575)||
|[334799](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334799)|dotnet/runtime|android-x64 Release AllSubsets_Mono_RuntimeTests minijit|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334799/logs/1059)|dotnet/runtime#88611|
|[336324](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336324)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  linux arm64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336324/logs/1523)|dotnet/runtime#88527|
|[336233](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336233)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336233/logs/1724)|dotnet/runtime#88711|
|[336279](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336279)|dotnet/runtime|iossimulator-arm64 Release AllSubsets_Mono_RuntimeTests |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336279/logs/649)|dotnet/runtime#87319|
|[336189](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336189)|dotnet/runtime|Build Tests|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336189/logs/2502)|dotnet/runtime#88064|
|[336248](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336248)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  linux arm64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336248/logs/489)|dotnet/runtime#88712|
|[336245](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336245)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  linux_musl x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336245/logs/1139)|dotnet/runtime#88712|
|[336064](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336064)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336064/logs/2437)|dotnet/runtime#88650|
|[335697](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335697)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335697/logs/1612)|dotnet/runtime#85694|
|[336024](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336024)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336024/logs/1586)|dotnet/runtime#88703|
|[335936](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335936)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335936/logs/2606)|dotnet/runtime#88245|
|[336100](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336100)|dotnet/runtime|Build Tests|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/336100/logs/2119)|dotnet/runtime#85328|
|[335908](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335908)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335908/logs/2423)|dotnet/runtime#88699|
|[335895](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335895)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335895/logs/2143)|dotnet/runtime#88626|
|[335899](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335899)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335899/logs/2550)|dotnet/runtime#88698|
|[335829](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335829)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335829/logs/2430)|dotnet/runtime#88695|
|[335821](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335821)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335821/logs/2455)|dotnet/runtime#88620|
|[335806](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335806)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335806/logs/2528)|dotnet/runtime#87562|
|[335579](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335579)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335579/logs/1632)|dotnet/runtime#88683|
|[335772](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335772)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335772/logs/1714)|dotnet/runtime#88669|
|[335781](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335781)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335781/logs/2467)|dotnet/runtime#88415|
|[335761](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335761)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335761/logs/1742)|dotnet/runtime#88604|
|[335841](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335841)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335841/logs/2714)||
|[335749](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335749)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335749/logs/2454)|dotnet/runtime#88537|
|[335753](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335753)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335753/logs/1619)|dotnet/runtime#88665|
|[335920](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335920)|dotnet/runtime|tvossimulator-x64 Release AllSubsets_Mono_RuntimeTests |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335920/logs/934)||
|[335728](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335728)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335728/logs/1580)|dotnet/runtime#88692|
|[335718](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335718)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335718/logs/2425)|dotnet/runtime#88634|
|[335714](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335714)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335714/logs/1764)|dotnet/runtime#88639|
|[335678](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335678)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335678/logs/1594)|dotnet/runtime#88457|
|[335673](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335673)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335673/logs/2606)|dotnet/runtime#88687|
|[335732](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335732)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  osx arm64 release|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335732/logs/930)|dotnet/runtime#88692|
|[335644](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335644)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run R2R_CG2 osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335644/logs/1363)||
|[335794](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335794)|dotnet/runtime|windows-x64 Checked NativeAOT |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335794/logs/1608)|dotnet/runtime#87579|
|[335750](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335750)|dotnet/runtime|ios-arm64 Release AllSubsets_Mono_RuntimeTests |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335750/logs/36)|dotnet/runtime#86089|
|[335637](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335637)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335637/logs/526)|dotnet/runtime#88623|
|[335638](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335638)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335638/logs/267)|dotnet/runtime#88623|
|[335641](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335641)|dotnet/runtime|Build Tests|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335641/logs/2261)|dotnet/runtime#88686|
|[334773](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334773)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334773/logs/2589)|dotnet/runtime#87717|
|[335676](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335676)|dotnet/runtime|iossimulator-arm64 Release AllSubsets_Mono_RuntimeTests |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335676/logs/773)|dotnet/runtime#88687|
|[335435](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335435)|dotnet/runtime|windows-x64 Checked NativeAOT |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335435/logs/1799)|dotnet/runtime#88623|
|[335636](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335636)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run R2R_CG2 linux x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335636/logs/1173)|dotnet/runtime#88623|
|[335171](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335171)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335171/logs/2633)|dotnet/runtime#88658|
|[335503](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335503)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335503/logs/2593)|dotnet/runtime#88245|
|[335457](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335457)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335457/logs/1769)|dotnet/runtime#88669|
|[335424](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335424)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335424/logs/1564)|dotnet/runtime#88665|
|[335353](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335353)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335353/logs/1486)|dotnet/runtime#88655|
|[335334](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335334)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335334/logs/1549)|dotnet/runtime#88661|
|[335352](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335352)|dotnet/runtime|iossimulator-x64 Release AllSubsets_Mono_RuntimeTests |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335352/logs/41)|dotnet/runtime#88655|
|[335295](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335295)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335295/logs/2186)|dotnet/runtime#88634|
|[333853](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333853)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/333853/logs/1786)|dotnet/runtime#88604|
|[335292](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335292)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335292/logs/2383)|dotnet/runtime#88650|
|[335275](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335275)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335275/logs/2341)|dotnet/runtime#88539|
|[335251](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335251)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335251/logs/2534)|dotnet/runtime#88245|
|[335193](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335193)|dotnet/runtime|Build Tests|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335193/logs/2340)|dotnet/runtime#88188|
|[335166](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335166)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335166/logs/1571)|dotnet/runtime#88657|
|[335087](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335087)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335087/logs/2559)|dotnet/runtime#88521|
|[335065](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335065)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335065/logs/2551)|dotnet/runtime#88520|
|[335161](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335161)|dotnet/runtime|tvossimulator-x64 Release AllSubsets_Mono_RuntimeTests |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335161/logs/41)|dotnet/runtime#88167|
|[335160](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335160)|dotnet/runtime|tvos-arm64 Release AllSubsets_Mono_RuntimeTests |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335160/logs/39)|dotnet/runtime#88167|
|[335050](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335050)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335050/logs/1465)|dotnet/runtime#88655|
|[335060](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335060)|dotnet/runtime|android-x64 Release AllSubsets_Mono_RuntimeTests minijit|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335060/logs/48)|dotnet/runtime#88655|
|[335062](https://dev.azure.com/dnceng-public/public/_build/results?buildId=335062)|dotnet/runtime|iossimulator-x64 Release AllSubsets_Mono_RuntimeTests |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/335062/logs/41)|dotnet/runtime#88655|
|[334986](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334986)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334986/logs/1716)|dotnet/runtime#88639|
|[334935](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334935)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334935/logs/2599)|dotnet/runtime#88245|
|[334926](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334926)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334926/logs/2370)|dotnet/runtime#88650|
|[334916](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334916)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334916/logs/2536)|dotnet/runtime#88268|
|[334911](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334911)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334911/logs/1576)|dotnet/runtime#88640|
|[334908](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334908)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334908/logs/1337)||
|[334885](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334885)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  linux x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334885/logs/528)|dotnet/runtime#85562|
|[334884](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334884)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run R2R_CG2 osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334884/logs/1250)|dotnet/runtime#85562|
|[334877](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334877)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334877/logs/2446)|dotnet/runtime#88620|
|[334907](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334907)|dotnet/runtime|linux-arm64 Release AllSubsets_Mono_LLVMFullAot_RuntimeTests llvmfullaot|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334907/logs/843)||
|[334862](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334862)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334862/logs/2577)|dotnet/runtime#87865|
|[334868](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334868)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334868/logs/2702)||
|[334508](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334508)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334508/logs/2343)|dotnet/runtime#85562|
|[334837](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334837)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334837/logs/1726)|dotnet/runtime#88464|
|[334820](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334820)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  linux_musl x64 release|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334820/logs/940)||
|[334821](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334821)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run R2R_CG2 osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334821/logs/414)||
|[333988](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333988)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/333988/logs/1698)|dotnet/runtime#88611|
|[334770](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334770)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  linux x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334770/logs/575)||
|[334454](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334454)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334454/logs/1630)|dotnet/runtime#88633|
|[334732](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334732)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334732/logs/2590)|dotnet/runtime#88245|
|[334689](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334689)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334689/logs/1603)|dotnet/runtime#88640|
|[334693](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334693)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334693/logs/2425)|dotnet/runtime#88641|
|[334631](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334631)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334631/logs/2517)|dotnet/runtime#88064|
|[334614](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334614)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run R2R_CG2 linux_musl x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334614/logs/1309)||
|[334587](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334587)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334587/logs/2483)|dotnet/runtime#87717|
|[334601](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334601)|dotnet/runtime|windows-x64 Checked NativeAOT |[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334601/logs/1474)|dotnet/runtime#88639|
|[334514](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334514)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334514/logs/2538)|dotnet/runtime#88415|
|[333574](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333574)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/333574/logs/2590)|dotnet/runtime#88521|
|[334461](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334461)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334461/logs/2378)|dotnet/runtime#88634|
|[334429](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334429)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334429/logs/2518)|dotnet/runtime#88632|
|[334355](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334355)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334355/logs/2023)|dotnet/runtime#88626|
|[334366](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334366)|dotnet/runtime|coreclr  Pri1 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334366/logs/520)|dotnet/runtime#88623|
|[334332](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334332)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334332/logs/2448)|dotnet/runtime#88433|
|[334367](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334367)|dotnet/runtime|osx-x64 Release AllSubsets_Mono_Minijit_RuntimeTests minijit|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334367/logs/1784)|dotnet/runtime#88623|
|[334257](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334257)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334257/logs/2441)|dotnet/runtime#88572|
|[334246](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334246)|dotnet/runtime|coreclr  Pri0 Runtime Tests Run  osx x64 checked|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/334246/logs/2734)||
Displaying 100 of 209 results

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[336407](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336407)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=336407&view=ms.vss-test-web.build-test-results-tab&runId=6988566&resultId=101930)||
|[334819](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334819)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334819&view=ms.vss-test-web.build-test-results-tab&runId=6941240&resultId=100320)||
|[334787](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334787)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334787&view=ms.vss-test-web.build-test-results-tab&runId=6941336&resultId=101124)||
|[334553](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334553)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=334553&view=ms.vss-test-web.build-test-results-tab&runId=6935116&resultId=112603)|dotnet/runtime#81993|
|[333312](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333312)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333312&view=ms.vss-test-web.build-test-results-tab&runId=6898998&resultId=100281)||
|[333303](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333303)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333303&view=ms.vss-test-web.build-test-results-tab&runId=6898796&resultId=102148)||
|[333109](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333109)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=333109&view=ms.vss-test-web.build-test-results-tab&runId=6892242&resultId=100185)|dotnet/runtime#88554|
|[332980](https://dev.azure.com/dnceng-public/public/_build/results?buildId=332980)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=332980&view=ms.vss-test-web.build-test-results-tab&runId=6888220&resultId=100305)||
|[332970](https://dev.azure.com/dnceng-public/public/_build/results?buildId=332970)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=332970&view=ms.vss-test-web.build-test-results-tab&runId=6888208&resultId=101115)||
|[332609](https://dev.azure.com/dnceng-public/public/_build/results?buildId=332609)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=332609&view=ms.vss-test-web.build-test-results-tab&runId=6877582&resultId=100277)||
|[332607](https://dev.azure.com/dnceng-public/public/_build/results?buildId=332607)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=332607&view=ms.vss-test-web.build-test-results-tab&runId=6877328&resultId=101118)||
|[331467](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331467)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331467&view=ms.vss-test-web.build-test-results-tab&runId=6855128&resultId=110495)|dotnet/runtime#88508|
|[331619](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331619)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331619&view=ms.vss-test-web.build-test-results-tab&runId=6850172&resultId=100279)||
|[331601](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331601)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331601&view=ms.vss-test-web.build-test-results-tab&runId=6850268&resultId=101116)||
|[331541](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331541)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331541&view=ms.vss-test-web.build-test-results-tab&runId=6849124&resultId=110495)|dotnet/runtime#88371|
|[331490](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331490)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331490&view=ms.vss-test-web.build-test-results-tab&runId=6847718&resultId=112752)|dotnet/runtime#88404|
|[331430](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331430)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331430&view=ms.vss-test-web.build-test-results-tab&runId=6847412&resultId=110495)|dotnet/runtime#88503|
|[331425](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331425)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331425&view=ms.vss-test-web.build-test-results-tab&runId=6847362&resultId=110499)|dotnet/runtime#88502|
|[331160](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331160)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331160&view=ms.vss-test-web.build-test-results-tab&runId=6844200&resultId=110495)|dotnet/runtime#87857|
|[331294](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331294)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331294&view=ms.vss-test-web.build-test-results-tab&runId=6844208&resultId=110499)|dotnet/runtime#88498|
|[331203](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331203)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331203&view=ms.vss-test-web.build-test-results-tab&runId=6841874&resultId=110495)|dotnet/runtime#86875|
|[331212](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331212)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331212&view=ms.vss-test-web.build-test-results-tab&runId=6840828&resultId=112415)||
|[331166](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331166)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331166&view=ms.vss-test-web.build-test-results-tab&runId=6841100&resultId=110495)|dotnet/runtime#87562|
|[331159](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331159)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331159&view=ms.vss-test-web.build-test-results-tab&runId=6840774&resultId=110495)||
|[331033](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331033)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331033&view=ms.vss-test-web.build-test-results-tab&runId=6837702&resultId=110495)|dotnet/runtime#86875|
|[331021](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331021)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=331021&view=ms.vss-test-web.build-test-results-tab&runId=6837546&resultId=110495)|dotnet/runtime#88368|
|[330843](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330843)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330843&view=ms.vss-test-web.build-test-results-tab&runId=6834342&resultId=110495)|dotnet/runtime#88466|
|[330822](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330822)|dotnet/runtime|[tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation/simpleruntimeeventvalidation.sh](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330822&view=ms.vss-test-web.build-test-results-tab&runId=6832160&resultId=107028)|dotnet/runtime#88480|
|[330743](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330743)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330743&view=ms.vss-test-web.build-test-results-tab&runId=6830626&resultId=110495)|dotnet/runtime#88467|
|[330680](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330680)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330680&view=ms.vss-test-web.build-test-results-tab&runId=6829302&resultId=112804)|dotnet/runtime#87771|
|[330572](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330572)|dotnet/runtime|[PayloadGroup0.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=330572&view=ms.vss-test-web.build-test-results-tab&runId=6826438&resultId=110499)|dotnet/runtime#87785|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|78|240|240|

**Comment by radical** [comment]
Looks like the test fails for non-wasm cases with a real error:
```
          0.8s: Expected:
          0.8s: {
          0.8s: "Microsoft-Windows-DotNETRuntime" = -1 +- -0
          0.8s: }
        
          0.8s: Actual:
          0.8s: {
          0.8s: "Microsoft-DotNETCore-EventPipe" = 1
          0.8s: }
```

I'll open a new PR to disable this.

**Comment by radical** [comment]
Keeping this issue open as the corresponding test is being *disabled* in https://github.com/dotnet/runtime/pull/88500, but still needs to be fixed. I will remove the `blocking-clean-ci` label once that PR is merged.

**Comment by lewing** [comment]
cc @lambdageek

**Comment by lambdageek** [comment]
@lateralusX  https://github.com/dotnet/runtime/issues/88499#issuecomment-1624462699 pretty sure this is expected right? we use EventPipe, not ETW.  The tests needs to be updated?

**Comment by kotlarmilos** [comment]
The pattern in the issue is causing false positives on PRs, so I will remove the `Known Build Error` label.

**Comment by LakshanF** [comment]
This test should not run on Mono runtime as @lateralusX mentioned above, and the linked issue is resolved. Closing this issue.
